pop-api/src/v0/assets/fungibles.rs

```diff
@@ -0,0 +1,408 @@
+use ink::prelude::vec::Vec;
+
+use crate::{assets, primitives::AssetId, AccountId, Balance, Result, StatusCode};
+
+/// Local Fungibles:
```

Comment:
Since it is unclear to me at the moment how interface might evolve based on the fact that we might want to add standards, the subject management of assets is unclear how it will be used (in constructor or not), I created an issue for this and will be tackled at a later point; #111